package/tests/test_api/test_graphql.py

```diff
@@ -513,3 +520,16 @@ def test_graphql_runs_tracking_data_endpoint(
             }
         }
         assert response.json() == expected_response
+
+
+class TestGraphQLSchema:
+    def testApolloSchema(self):
+        here = path.abspath(
+            path.join(path.dirname(__file__), "..", "..", "..", "src", "apollo")
+        )
+        changes = []
+        with open(path.join(here, "schema.graphql")) as data:
+            apollo_schema = data.read()
+            changes = diff(apollo_schema, str(schema))
+            print_diff(changes)
+        assert changes == []
```

Comment:
This coul dbe done with 
```suggestion
            changes = diff(apollo_schema, str(schema))
        assert not changes, print_diff(changes)
```
This only prints if the assert fails.